v1/src/test/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScannerIT.java

```diff
@@ -890,4 +893,35 @@ public void pgSequences() throws Exception {
             + "\n\tPRIMARY KEY (\"id\")\n)\n\n";
     assertThat(ddl.prettyPrint(), equalToCompressingWhiteSpace(expectedDdl));
   }
+
+  @Test
+  public void protobundle() throws Exception {
+    List<String> statements =
+        Arrays.asList(
+            "CREATE PROTO BUNDLE ("
+                + "com.google.cloud.teleport.spanner.tests.TestMessage,"
+                + "com.google.cloud.teleport.spanner.tests.TestEnum)",
+            "CREATE TABLE `Account` ("
+                + " `id`        INT64 ,"
+                + " `balanceId` INT64 NOT NULL,"
+                + " ) PRIMARY KEY (`id` ASC)");
+
+    FileDescriptorSet.Builder fileDescriptorSetBuilder = FileDescriptorSet.newBuilder();
+    fileDescriptorSetBuilder.addFile(
+        com.google.cloud.teleport.spanner.tests.TestMessage.getDescriptor().getFile().toProto());
+    ByteString protoDescriptorBytes = fileDescriptorSetBuilder.build().toByteString();
+
+    spannerServer.createDatabase(dbId, statements, protoDescriptorBytes);
+    Ddl ddl = getDatabaseDdl();
+    assertEquals(FileDescriptorSet.parseFrom(protoDescriptorBytes), ddl.protoDescriptors());
+    String expectedDdl =
+        "\nCREATE PROTO BUNDLE ("
+            + "\n\tcom.google.cloud.teleport.spanner.tests.TestMessage, "
+            + " com.google.cloud.teleport.spanner.tests.TestEnum)"
+            + "CREATE TABLE `Account` ("
+            + "\n\t`id`                                    INT64,"
+            + "\n\t`balanceId`                             INT64 NOT NULL,"
+            + "\n) PRIMARY KEY (`id` ASC)\n\n";
```

Comment:
```suggestion
        "\nCREATE PROTO BUNDLE ("
            + " com.google.cloud.teleport.spanner.tests.TestMessage, "
            + " com.google.cloud.teleport.spanner.tests.TestEnum)"
            + "CREATE TABLE `Account` ("
            + "  `id`                                    INT64,"
            + "  `balanceId`                             INT64 NOT NULL,"
            + ") PRIMARY KEY (`id` ASC)";
```

Reply:
The helper method doesn't compress \n and \t as whitespace characters. So, keeping as is.